packages/digitallibrary/templates/scarc/digitalcontent.inc.php

```diff
@@ -0,0 +1,290 @@
+      {
+         if($objFile->DefaultAccessLevel == DIGITALLIBRARY_ACCESSLEVEL_PREVIEWONLY)
+         {
+            echo("<span class='bold'>NOTE</span>: The public cannot download this file.<br/>");
+         }
+         elseif($objFile->DefaultAccessLevel == DIGITALLIBRARY_ACCESSLEVEL_NONE)
+         {
+            echo("<span class='bold'>NOTE</span>: The public cannot view previews of or download this file.<br/>");
+         }
+      }
+
+      if(!$PreviewAccess)
+      {
+         echo("<span class='digcontentfiletitle'>No preview for this item is publicly available. Contact the archives for information about accessing this item.<br/><br/>");
+         echo("{$objFile->getString('Title')} (" . ($objFile->FileType ? $objFile->FileType->getString('FileType') : '') . ", " . formatsize($objFile->Size) . ")</span><br/>");
+      }
+      else
+      {
+         if(encoding_substr_count($objFile->FileType->FileExtensions, '.pdf') && file_exists("{$_ARCHON->PublicInterface->ImagePath}/pdficon_large.gif"))
+         {
+            $onclick = ($_ARCHON->config->GACode && $_ARCHON->config->GADigContentPrefix) ? "onclick='javascript: pageTracker._trackPageview(\"{$_ARCHON->config->GADigContentPrefix}/pdf/DigitalContentID={$objDigitalContent->ID}/fileID={$objFile->ID}\");'" : "";
+            echo("<img src='{$_ARCHON->PublicInterface->ImagePath}/pdficon_large.gif' alt='PDF icon' /><br/>");
+         }
+         elseif($objFile->FileType->MediaType->MediaType == 'Image')
+         {
+            $onclick = ($_ARCHON->config->GACode && $_ARCHON->config->GADigContentPrefix) ? "onclick='javascript: pageTracker._trackPageview(\"{$_ARCHON->config->GADigContentPrefix}/image/DigitalContentID={$objDigitalContent->ID}/fileID={$objFile->ID}\");'" : "";
+            echo("<img class='digcontentfile' src='".$objFile->getFileURL(DIGITALLIBRARY_FILE_PREVIEWLONG)."' alt='{$objFile->getString('Title')}'/><br/>");
+         }
+         elseif($objFile->Filename)
+         {
+            preg_match("/.+?\/(.+)/u", $objFile->FileType->getString('ContentType'), $matches);
+            $contenttype = $matches[1] ? $matches[1] : 'file';
+            $onclick = ($_ARCHON->config->GACode && $_ARCHON->config->GADigContentPrefix) ? "onclick='javascript: pageTracker._trackPageview(\"{$_ARCHON->config->GADigContentPrefix}/{$contenttype}/DigitalContentID={$objDigitalContent->ID}/fileID={$objFile->ID}\");'" : "";
+            echo("<script type='text/javascript'>embedFile($objFile->ID, '" . encode($objFile->FileType->MediaType->MediaType, ENCODE_JAVASCRIPT) . "', 'long');</script><br/>");
+         }
+
+         echo("<span class='digcontentfiletitle'><strong>Description:</strong> " . $objFile->getString('Title'). "<br/>");
+
+         if($FullAccess)
+         {
+//            echo("<a href='?p=digitallibrary/getfile&amp;id=$objFile->ID' $onclick>Download Original File</a><br/>");

```

Comment:
Again, this is another case of "commented it out until we figure out what to do with it"